esp/clients/ws_dfsclient/ws_dfsclient.cpp

```diff
@@ -556,16 +561,64 @@ IClientWsDfs *getDfsClient(const char *serviceUrl, IUserDescriptor *userDesc)
     return dfsClient.getClear();
 }
 
+static CriticalSection localSecretCrit;
+static void configureClientSSL(IEspClientRpcSettings &rpc, const char *secretName)
+{
+    /*
+     * This is a bit of a kludge, it gets the certificates from secrets, and writes them to local temp strorage.
+     * It does this so that it can pass the filename paths to rpc ssl / secure socket layer, which currently only
+     * accepts filenames, not binary blobs from memory.
+     */
```

Comment:
@afishbeck - was a JIRA opened for securesocket for the work to get it to support certs/keys as in-memory blobs?

Reply:
Yes, https://track.hpccsystems.com/browse/HPCC-27255.  It's a sub task of the JIRA for getting TLS secrets from a hashicorp vault.